packages/plugin-core/src/_extension.ts

```diff
@@ -653,9 +651,23 @@ export async function _activate(
           .vaults.filter(VaultUtils.isSelfContained).length,
         numRemoteVaults: ws.getDWorkspace().vaults.filter(VaultUtils.isRemote)
           .length,
-        numContributors: _.max(await numContributors),
       };
 
+      // Track contributors to repositories, but do so in the background so
+      // initialization isn't delayed.
+      const startGetAllReposNumContributors = process.hrtime();
+      wsService
+        .getAllReposNumContributors()
+        .then((numContributors) => {
+          AnalyticsUtils.track(GitEvents.ContributorsFound, {
+            maxNumContributors: _.max(numContributors),
+            duration: getDurationMilliseconds(startGetAllReposNumContributors),
```

Comment:
don't need to do this now but moving forward, might make sense to shift tracking performance related metrics to sentry since they deliberately do apm (application performance monitoring) which amplitude is not great at. an async for another time